components/Buttons/RoundButton.js

```diff
@@ -6,13 +6,17 @@ export default function RoundButton({
   variant,
   isRotate,
   children,
+  disabled,
+  grayout,
 }) {
   return (
     <StyledRoundButton
       type={type}
       onClick={onClick}
       $variant={variant}
       $isRotate={isRotate}
+      disabled={disabled}
+      $grayout={grayout}
```

Comment:
is there a scenario where you gray out the button but do not disable it? If not you can reuse the "disabled" prop for the grayout styling.

Reply:
Yes, the button with the plus icon that toggles the form should be disabled but not opaque (which I do with the grayout prop). That's why I have two different props here. If I did both with one prop for these buttons, I would have to use another one for the form-toggle-button and would then have the code repetition in the RoundButton component. This seemed to me to be the better solution. Do you see it differently?